tix_clock/tix_clock.ino

```diff
@@ -7,19 +7,27 @@ const int LATCH_PIN = 3;
 const int CLOCK_PIN = 4;
 const int NUMBER_OF_OUTPUT_PINS = 3;
 int OUTPUT_PINS[] = {DATA_PIN, LATCH_PIN, CLOCK_PIN};
+int hours = 25;
```

Comment:
Why hours is set to 25? Please, just declare the variable without assigning it a value.